lighthouse-core/audits/metrics/cumulative-layout-shift.js

```diff
@@ -46,9 +46,9 @@ class CumulativeLayoutShift extends Audit {
     return {
       // Calibrated to assure 0.1 gets a score of 0.9. https://web.dev/cls/#what-is-a-good-cls-score
       // This 0.1 target score was determined through both manual evaluation and large-scale analysis.
-      // see https://www.desmos.com/calculator/wmcxn7zfhc
-      scorePODR: 0.02,
-      scoreMedian: 0.2,
+      // see https://www.desmos.com/calculator/vjb652sqeh
```

Comment:
seems like it didn't save with the new values, maybe just forgot to press save?

Reply:
![image](https://user-images.githubusercontent.com/39191/80433467-dd314400-88ab-11ea-8f31-86334d10bb5a.png)


true. just updated with the newer shiny one.